addons_rksv/pos_rksv/views/res_company.xml

```diff
@@ -23,10 +26,21 @@
                         </group>
                     </page>
                 </notebook>
+            </field>
+        </record>
+
+        <!-- TODO -->
```

Comment:
Why not add the base_vat dependency? What changed your mind to make it possible to inherit like this as there seemed to be an issue in 11.0?